internal/templates/file.html

```diff
@@ -27,7 +27,7 @@
     SyntaxHighlighter.autoloader(
     ['essence','param','solution', '{{prefix_path}}/js/shBrushEssence.js'],
     ['essenceprime', '{{prefix_path}}/js/shBrushEssencePrime.js'],
-    ['minizinc','mzn', '{{prefix_path}}/js/shBrushMiniZinc.js'],
+    ['minizinc', '{{prefix_path}}/js/shBrushMiniZinc.js'],
     ['comet','co', '{{prefix_path}}/js/shBrushComet.js'],
```

Comment:
.co is the standard one.  .comet can be removed.

On Monday, September 7, 2015, Bilal Syed Hussain <notifications@github.com>
wrote:

> In internal/templates/file.html
> <https://github.com/csplib/csplib/pull/109#discussion_r38877076>:
>
> > @@ -27,7 +27,7 @@
> >      SyntaxHighlighter.autoloader(
> >      ['essence','param','solution', '{{prefix_path}}/js/shBrushEssence.js'],
> >      ['essenceprime', '{{prefix_path}}/js/shBrushEssencePrime.js'],
> > -    ['minizinc','mzn', '{{prefix_path}}/js/shBrushMiniZinc.js'],
> > +    ['minizinc', '{{prefix_path}}/js/shBrushMiniZinc.js'],
> >      ['comet','co', '{{prefix_path}}/js/shBrushComet.js'],
>
> Why does comet have two extensions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/csplib/csplib/pull/109/files#r38877076>.
>